src/components/nodes/node.js

```diff
@@ -18,6 +26,26 @@ export default class Node {
     return this.type === type;
   }
 
+  isStartGroup() {
```

Comment:
Minor thing: I'm wondering if the concept of "group" will throw us later when we've forgotten the discussions around this. Would a clunkier but more specific name like `isStartEvent` or so work for you?